working-groups/meta/rfc-drafts/compiler-team-contributors.md

```diff
@@ -0,0 +1,244 @@
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Introduce an intermediate level of member for the compiler team, the
+**compiler team contributor**.
+
+# Motivation
+[motivation]: #motivation
+
+This proposal is part of a larger effort to introduce more structure
+into the compiler team's makeup. This structure should make it easier
+to become a part of the compiler team, by laying out a clear path to
+membership and by offering more official roles.
+
+## Background: Access to infrastructure
+
+In addition to recognition, the levels in this document control access
+to other bits of our infrastructure. It is worth spending some time
+reviewing those bits of infrastructure.
+
+### bors r+ privileges
+
+The bors bot has a central list of folks that have "r+" privileges.
+These are people who can leave comments instructing bors to land a PR.
+
+While the bors permissions are very crude (you either have privileges
+or you don't), we have historically asked people to use their
+permissions in specific ways (self-policed).
+
+One reason that it is convenient to have r+ privileges is a purely
+adminstrative one: they allow you to re-approve PRs after they have
+been rebased, which is a common need. (Typically this is done by
+writing `@bors r=username`, where `username` is the name of the
+original reviewer.)
+
+Apart from these adminstrative re-reviews, the expectation is that
```

Comment:
adminstrative -> administrative

Reply:
```suggestion
Apart from these administrative re-reviews, the expectation is that
```
(again, makes applying this change a bit quicker)